searchlib/src/vespa/searchlib/queryeval/sourceblendersearch.cpp

```diff
@@ -158,14 +158,14 @@ SourceBlenderSearch::setChild(size_t index, SearchIterator::UP child) {
     _sources[_children[index]] = child.release();
 }
 
-SourceBlenderSearch *
+SearchIterator::UP
 SourceBlenderSearch::create(std::unique_ptr<sourceselector::Iterator> sourceSelector,
                             const Children &children, bool strict)
 {
     if (strict) {
-        return new SourceBlenderSearchStrict(std::move(sourceSelector), children);
+        return SearchIterator::UP(new SourceBlenderSearchStrict(std::move(sourceSelector), children));
```

Comment:
Should you not use std::make_unique instead ?

Reply:
I wanted to handle Strict and non-Strict the same way, and the other constructor wasn't public; will adjust with a NonStrict variant and use make_unique.